# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/graphite2/no-tests.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- graphite2-1.3.14/CMakeLists.txt.vanilla	2025-04-27 22:33:16.734819274 +0300
+++ graphite2-1.3.14/CMakeLists.txt	2025-04-27 22:33:21.468375594 +0300
@@ -82,7 +82,7 @@
 endif()
 
 add_subdirectory(src)
-add_subdirectory(tests)
+#add_subdirectory(tests)
 add_subdirectory(doc)
 if (NOT GRAPHITE2_NFILEFACE)
     add_subdirectory(gr2fonttest)
